docs/index.html

```diff
@@ -0,0 +1,252 @@
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="hover:text-rose-700">
+              <a href="about.html">About</a>
+            </li>
+
+            <li class="hover:text-rose-700">
+              <a href="#program">Program</a>
+            </li>
+
+            <li class="hover:text-rose-700">
+              <a href="#join">Join</a>
+            </li>
+
+            <li class="hover:text-rose-700">
+              <a href="#sponsor">Sponsor</a>
+            </li>
+
+            <li class="hover:text-rose-700">
+              <a href="#news">News</a>
+            </li>
+          </ul>
+        </nav>
+
+        <a href="#home">
+          <h3 class="border-4 px-4 py-2 border-rose-700 text-rose-700 font-bold hover:bg-rose-700 hover:text-white hover:shadow-md hover:shadow-zinc-500 ease-in-out delay-200">
+            Garden Campaign
+          </h3>
+        </a>
+      </div>
+
+      <!-- Mobile Menu -->
+
+      <div class="md:hidden bg-hero-image bg-center bg-no-repeat bg-cover bg-blend-overlay bg-stone-300 px-5 fixed top-0 left-0 right-0 h-[100vh] transition translate-x-full delay-100 target:-translate-x-0" id="menu">
+        <div class="md:hidden flex justify-between content-center flex-wrap pb-[50px] py-3">
+          <a href="#home">
+            <img src="./images/flower-logo1.png" alt="Logo" width="150px" class="block">
+          </a>
+
+          <a href="#home">
+            <i class="fa-solid fa-x fa-xl"></i>
+          </a>
+        </div>
+
+        <nav>
+          <ul class="flex flex-col gap-5 text-3xl font-bold px-6">
+            <li>
+              <a href="index.html">Home</a>
+            </li>
+
+            <li>
+              <a href="about.html">About</a>
+            </li>
+
+            <li>
+              <a href="#program">Program</a>
+            </li>
+
+            <li>
+              <a href="#join">Join</a>
+            </li>
+
+            <li>
+              <a href="#sponsor">Sponsor</a>
+            </li>
+
+            <li>
+              <a href="#news">News</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <!-- Hero Section -->
+
+      <section class="bg-hero-image bg-center bg-no-repeat bg-cover bg-blend-overlay bg-stone-300 pb-[100px] px-4 py-3 md:px-[5%] lg:px-[15%] md:py-[100px]">
+
+        <div class="md:hidden flex justify-between content-center flex-wrap pb-[50px]">
+          <a href="#home">
+            <img src="./images/flower-logo1.png" alt="Logo" width="150px" class="block">
+          </a>
+
+          <a href="#menu">
+            <i class="fa-solid fa-bars fa-xl"></i>
+          </a>
+        </div>
+
+        <h2 class="text-3xl text-emerald-800 mb-2">
+          "Hello, Sharing world!"
+        </h2>
+
+        <h1 class="text-6xl font-extrabold text-emerald-800/[0.8] mb-6 lg:w-[70%] md:text-8xl">
+          Garden and Flower Show 2023
+        </h1>
```

Comment:
Please make sure to follow the guidelines as requested by the previous reviewer 🤓 the project requirement clearly states that you should stick to the colors 👇🏽 

![Screenshot 2023-04-21 at 01 36 41](https://user-images.githubusercontent.com/45739963/233549059-e5887177-0d31-4df7-9e3b-ca1ddd67e1b9.png)

You did a great job on the Layout and personalizing the content but make sure to also follow the color swatch as it is part of the project requirements, the reviewer doesn't make the project requirements, so if you have any concerns about the curriculum you can propose some changes or reach out to student success. 👨🏽‍🎓 

NB: Make sure to address this issue everywhere the previous reviewer pointed out. 💯